privacy-profile-composer/pkg/envoyfilter/internal/common/pii_analysis.go

```diff
@@ -0,0 +1,60 @@
+package common
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+)
+
+func PiiAnalysis(presidioSvcURL string, svcName string, bufferBytes []byte) (string, error) {
```

Comment:
is this function being used somewhere or did you just move it here from `inbound_filter.go` file?

Reply:
it's being called in the DecodeData function